pass-secret-service doesn't build

### Describe the bug

`nix-build -A pass-secret-service` doesn't work anymore. The tests show errors and take forever (or are stuck).

### Steps To Reproduce
Steps to reproduce the behavior:
1. switch to `master` branch in `nixpkgs`
2. `nix-build -A pass-secret-service`

### Expected behavior

The build should finish successfully.

### Screenshots

```
running install tests
===== Preparing gpg test keys in test/.gnupghome =====
mkdir -p -m 700 test/.gnupghome
gpg --allow-secret-key-import --import test/test_key.asc
gpg: keybox '/build/source/test/.gnupghome/pubring.kbx' created
gpg: /build/source/test/.gnupghome/trustdb.gpg: trustdb created
gpg: key CF3AF3078C2A59A7: public key "pass_secret_service test key (INSECURE TEST KEY)" imported
gpg: key CF3AF3078C2A59A7: secret key imported
gpg: Total number processed: 1
gpg:               imported: 1
gpg:       secret keys read: 1
gpg:   secret keys imported: 1
gpg --import-ownertrust test/test_ownertrust.txt
gpg: inserting ownertrust of 6
===== Preparing password store in test/.test-password-store =====
pypass init -p test/.test-password-store "8c2a59a7"
Password store initialized for 8c2a59a7.
dbus-run-session --config-file=/nix/store/6bg6qs8r26c044xgkzqyk4d0im81sa02-dbus-1.12.20/share/dbus-1/session.conf -- pytest -v test
============================= test session starts ==============================
platform linux -- Python 3.9.6, pytest-6.2.5, py-1.10.0, pluggy-1.0.0 -- /nix/store/8dxxjbiyxwkvh53q5kh6nydla2anacgi-python3-3.9.6/bin/python3.9
cachedir: .pytest_cache
rootdir: /build/source
plugins: asyncio-0.15.1
collected 23 items                                                             

test/test_collection.py::TestCollection::test_create_delete_collection ERROR [  4%]
test/test_collection.py::TestCollection::test_properties ERROR           [  8%]
test/test_collection.py::TestCollection::test_lock_unlock ERROR          [ 13%]
test/test_collection.py::TestCollection::test_persisted_item FAILED      [ 17%]
test/test_items.py::TestCollection::test_create_delete_item ERROR        [ 21%]
test/test_items.py::TestCollection::test_get_and_set_secret ERROR        [ 26%]
test/test_items.py::TestCollection::test_item_properties ERROR           [ 30%]
test/test_items.py::TestCollection::test_broken_session_path ERROR       [ 34%]
test/test_items.py::TestCollection::test_item_lookup ERROR               [ 39%]
test/test_items.py::TestCollection::test_item_in_deleted_collection ERROR [ 43%]
test/test_items.py::TestCollection::test_replace_item ERROR              [ 47%]
```

### Additional context
Add any other context about the problem here.

### Notify maintainers
<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

/cc @jluttine 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.14.14-200.fc34.x86_64, Fedora, 34 (Workstation Edition)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.16`
 - channels(root): `""`
 - nixpkgs: `/home/cassou/Documents/projects/nix-system/nixpkgs`
```

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

ran checkPhase in `nix develop` and cancelled the check when it hung and got these
```
========================================================== short test summary info ==========================================================
FAILED test/test_collection.py::TestCollection::test_persisted_item - Exception: could not find .gpg-id file
ERROR test/test_collection.py::TestCollection::test_create_delete_collection - Exception: could not find .gpg-id file
ERROR test/test_collection.py::TestCollection::test_properties - Exception: could not find .gpg-id file
ERROR test/test_collection.py::TestCollection::test_lock_unlock - Exception: could not find .gpg-id file
ERROR test/test_items.py::TestCollection::test_create_delete_item - Exception: could not find .gpg-id file
ERROR test/test_items.py::TestCollection::test_get_and_set_secret - Exception: could not find .gpg-id file
ERROR test/test_items.py::TestCollection::test_item_properties - Exception: could not find .gpg-id file
ERROR test/test_items.py::TestCollection::test_broken_session_path - Exception: could not find .gpg-id file
ERROR test/test_items.py::TestCollection::test_item_lookup - Exception: could not find .gpg-id file
ERROR test/test_items.py::TestCollection::test_item_in_deleted_collection - Exception: could not find .gpg-id file
ERROR test/test_items.py::TestCollection::test_replace_item - Exception: could not find .gpg-id file
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! KeyboardInterrupt !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

--- a/pkgs/applications/misc/pass-secret-service/default.nix
+++ b/pkgs/applications/misc/pass-secret-service/default.nix
@@ -19,9 +19,8 @@ python3.pkgs.buildPythonApplication rec {
   # /etc/ in check phase.
   postPatch = ''
     substituteInPlace Makefile \
-      --replace \
-        "dbus-run-session" \
-        "dbus-run-session --config-file=${dbus}/share/dbus-1/session.conf"
+      --replace "dbus-run-session" "dbus-run-session --config-file=${dbus}/share/dbus-1/session.conf" \
+      --replace '-p $(relpassstore)' '-p $(PASSWORD_STORE_DIR)'
   '';
 
   propagatedBuildInputs = with python3.pkgs; [

